lib/firebase/fetch_survey.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+// Data model for a survey question
+class SurveyQuestion {
```

Comment:
We already have a [SurveyQuestion class](https://github.com/theheartcenter-one/thc/blob/645cb280fad95074fcb40296267e85b6777505e0/lib/home/surveys/survey_questions.dart#L25), so eventually we'll either change this class name or combine the two classes.